### DISCUSSION TITLE
Can Chat on this page work for pdfs?

### DISCUSSION THREAD

**Discussion by zono50** [n4ze3m/page-assist#220] [Q&A]
I work with a lot of technical documents, and i like to be able to ask my documents questions, but whenever i try on a pdf, it just freezes and won't do anything until i close and reopen it. Regular page searches work tho.

This is using Arch linux with floorp browser

**Comment by n4ze3m** [comment]
Hey, sorry about the issue. I didn't test the extension with the Floorp browser, but I'll look into what happens and let you know

**Reply by zono50** [reply]
I appreciate you looking into this for me.

**Reply by zono50** [reply]
Were you able to check on this for me?

**Reply by n4ze3m** [reply]
Sorry for the late reply @zono50 . I checked, and the issue with chatting with PDFs is caused by the PDF parser we use not working on Firefox. This will be fixed in this weekend's release. 

Also, you can enable chatting on the website with larger content by using embedding model.

I opened this as issue #228 and will inform you when a fix is released.

https://github.com/user-attachments/assets/76613e27-47a9-47b5-9475-dfc4aaf7726f
